packages/language-typescript/src/index.ts

```diff
@@ -89,7 +89,7 @@ const fixDescriptionDocblock: (description?: string) => string | undefined = des
 
 export const DEFAULT_DOCUMENTATION_GENERATOR: GenerateDocumentation = ({ description, tags = [] }) => (description || tags.length) ? `
   /**
-   * ${filterAndJoinArray([fixDescriptionDocblock(description), ...tags.map(({ tag, value }) => `@${tag} ${value}`)], '\n* ')}
+   * ${filterAndJoinArray([fixDescriptionDocblock(description), ...tags.map(({ tag, value }) => `@${tag} ${typeof value === 'object' ? JSON.stringify(value) : value}`)], '\n* ')}
```

Comment:
I wonder if we should just JSON.stringify everything, so then strings will be quoted

Reply:
I've add object check for it. But i don't know how fix snapshot.

Reply:
what I meant was that I think we should just JSON.stringify instead of having the typecheck.

you should be able to fix the snapshot by running `npm test -- -u`